lib/github_api/error/service_error.rb

```diff
@@ -39,6 +41,9 @@ def initialize(response)
         @body    = response[:body]
         @status  = response[:status]
 
+        @response_headers = @headers
+        @response_message = @body
```

Comment:
These were already exposed from the ```GithubError``` class that ```ServiceError``` inherits from. They were nil. It may be beneficial for some somebody to see the raw header and body. If not, then I recommend that the ```response_message``` and ```response_header``` should not be exposed. https://github.com/jonmchan/github/blob/939d1e303e4da8201d77ac43381927a4450ce964/lib/github_api/error.rb#L10